brain/neuron/morphologyImpl.cpp

```diff
@@ -76,47 +47,20 @@ Morphology::Impl::Impl(const brion::Morphology& morphology)
     somaSection = ids[0];
 }
 
-bool Morphology::Impl::_fromBinary(const void* data, const size_t size)
+Morphology::Impl::Impl(const brion::Morphology& morphology)
+    : brion::plugin::MorphologyZeroEQ(morphology)
```

Comment:
This inheritance doesn't neither look right to me at all.
First of all, ZeroEQ is an optional dependency. Given that, making a core class of Brain depend on something that has the ZeroEQ suffix doesn't only smell fishy, is already rotten. Maybe it's just the naming because I haven't reached the class declaration yet, but it could also be that inheritance is used where it should be aggregation.